packages/core/src/Popover/Popover.tsx

```diff
@@ -0,0 +1,159 @@
+  align: 'start' | 'end' | 'center'
+} {
+  const match = /([a-z]+)-?([a-z]+)?/.exec(placement)
+  if (match) {
+    return {
+      // @ts-ignore
+      side: match[1],
+      // @ts-ignore
+      align: match[2],
+    }
+  }
+}
+
+const DsPopover = ({
+  borderRadius = 'xl',
+  children,
+  color = 'background.lightest',
+  hideArrow,
+  hideOverlay = false,
+  idx,
+  isOpen,
+  onClose,
+  onOpen,
+  openOnMount,
+  openOnHover,
+  overlayOpacity = 0.5,
+  p,
+  placement = 'top',
+  toggleIsOpenOnClick = true,
+  renderContent: RenderContent,
+  width,
+}: PopoverProps) => {
+  const [isOpenState, setIsOpenState] = React.useState(openOnMount || isOpen)
+
+  React.useEffect(() => {
+    if (isOpen !== isOpenState) {
+      setIsOpenState(isOpen)
+    }
+  }, [isOpen])
+
+  function handleOnOpenChange(open: boolean) {
+    setIsOpenState(open)
+    if (open) {
+      onOpen?.()
+    } else {
+      onClose?.()
+    }
+  }
+
+  function handleTriggerClick(event) {
+    if (!toggleIsOpenOnClick) {
+      event.preventDefault()
+    }
+  }
+
+  function handleMouseEnter() {
+    if (openOnHover) {
+      setIsOpenState(true)
+    }
+  }
+
+  function handleMouseLeave() {
+    if (openOnHover) {
+      setIsOpenState(false)
+    }
+  }
+
+  const { side, align } = parsePlacement(placement)
+
+  return (
+    <Popover.Root open={isOpenState} onOpenChange={handleOnOpenChange} defaultOpen={openOnMount}>
+      <Popover.Trigger
+        asChild
+        onClick={handleTriggerClick}
+        onMouseEnter={handleMouseEnter}
+        onMouseLeave={handleMouseLeave}
+      >
+        {children}
+      </Popover.Trigger>
+      <Popover.Portal>
+        {/* How can we wrap the content in a DS theme provider? */}
+        <Popover.Content sideOffset={5} side={side} align={align} id={idx}>
+          <ThemeProvider>
+            <Box color={color} boxShadowSize='overlay-md' borderRadius={borderRadius} p={p} width={width}>
+              <RenderContent />
+
+              <FixedCloseButton>
+                {/* @ts-ignore */}
+                <CloseButton as={Popover.Close} aria-label='Close' />
+              </FixedCloseButton>
+              {!hideArrow && <StyledArrow as={Popover.Arrow} color={color} />}
+            </Box>
+          </ThemeProvider>
+        </Popover.Content>
+      </Popover.Portal>
+      {!(hideOverlay || openOnHover) && <Overlay overlayOpacity={overlayOpacity} isOpen={isOpenState} />}
+    </Popover.Root>
+  )
+}
+
+// Old Popover props todo:
+// onBeforeOpen: PropTypes.func,
+// onBeforeClose: PropTypes.func,
+
+// Currently broken in plcn-popover:
+// borderColor: PropTypes.string,
+
+// Is this still needed?
+// display: PropTypes.string,
+// zIndex: PropTypes.number, <-- handled automatically by radix
+// trapFocus: PropTypes.bool, <-- fully managed by radix
+// stopPropagation: PropTypes.bool,
+// disableFloating: PropTypes.bool, <-- this controlled 'flip' in the old useFloating hook
```

Comment:
Is this all needed still?

Reply:
I think `disableFloating` may still have some utility, but I'm unsure about the rest.

Reply:
i'd start with none of it, most of it is probably intrinsic.  we can fill in gaps as they come up